release_notes.md

```diff
@@ -1,21 +1,10 @@
 # What's New
 
-## Introducing Top SOC Questions
-
-Edit or disable existing prompts or add new prompts based on your security posture.
-
-- You can now customize your interactions with the LLM by providing specific details and guidelines for your chats. Define your SOC questions to customize your LLM interactions.
-
-- Specific modules now have pre-defined questions. For example, under the **Alerts** module, questions include:
-
-    - Get Alert Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
-    
-    Similarly, for the **Incident** module, questions may be:
-    
-    - Get Incident Summary
-    - Generate Response Recommendations
-    - Generate Investigation Report
-    - MITRE ATT&CK Insight
+- The AI Assistant can understand and respond to context sensitive conversations and perform given SOC tasks
+    - Through enhanced comprehension, the bot can navigate to modules, its records, and playbook logs
+    - Sensitive data is being actively masked for all interactions with the API, considering the various modules to which the bot has access
+    - The bot can now generate playbooks for all applicable modules and now has access to create and update records
```

Comment:
The bot can now generate playbooks for all applicable modules --> this is present from previous release. It shouldn't be part of RN

Reply:
and now has access to create and update record --> this can be added as usecase and such use-cases cab added too which are supported by FrotiAI as part of conversation